features/product/viewing_products/viewing_product_from_specific_taxon_sorted_by_position.feature

```diff
@@ -6,16 +6,26 @@ Feature: Viewing products from specific taxon sorted by position
 
     Background:
         And the store operates on a channel named "Poland"
-        And the store classifies its products as "Soft Toys"
+        And the store classifies its products as "Toys"
+        And the "Toys" taxon has child taxon "Soft Toys"
         And the store has a product "Old pug"
-        And this product is in "Soft Toys" taxon at 1st position
-        And the store has a product "Small pug"
+        And this product is in "Toys" taxon at 1st position
         And this product is in "Soft Toys" taxon at 3rd position
+        And the store has a product "Small pug"
+        And this product is in "Toys" taxon at 3rd position
+        And this product is in "Soft Toys" taxon at 1st position
         And the store has a product "Young pug"
+        And this product is in "Toys" taxon at 2nd position
         And this product is in "Soft Toys" taxon at 2nd position
 
-    @ui @api
```

Comment:
Does it not work in both scenarios with the `@api` tag?

Reply:
only with ui